SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriverDebugContext.cpp

```diff
@@ -439,6 +439,8 @@ void USpatialNetDriverDebugContext::TickServer()
 				{
 					FWorkerComponentData CompData = Data.Component.CreateDebugComponent();
 					NetDriver->Connection->SendAddComponent(Entity, &CompData);
+					NetDriver->Connection->GetCoordinator().RefreshEntityCompleteness(Entity);
```

Comment:
Not sure this does anything, as the subview that cares about this component is still waiting on the tag from Interest due to this component being added

Reply:
So this only happens after an actor is ready, meaning we would have got the actor back from the runtime, with the debug tag.
So the way I see it, it is in the tagged entities, but did not pass the filter ?

Reply:
That is true, I'll separate the queries/